frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -179,13 +179,13 @@
             parameters.url = 'participants/participant_team';
             parameters.method = 'POST';
             parameters.data = {
-                "team_name": vm.team.name,
-                "team_url": vm.team.url
+                "team_name": vm.new.team.name,
```

Comment:
Can you please change `vm.new.team.name` to `vm.createNewTeam.name` and `vm.createNewTeam.url`

Reply:
While changing `vm.new.team.name` to `vm.createNewTeam.name` it is throwing error
`TypeError: Cannot assign to read only property 'name' of function 'function() {
            vm.isExistLoader = true;
            vm.loaderTitle = '';
            vm.loaderContain...<omitted>... }'`
@RishabhJain2018 Should I need to change something else

Reply:
I think createNewTeam is the function name in the controller so we cannot use that name again as a variable name. @RishabhJain2018